flatsurf/geometry/surface.py

```diff
@@ -1021,16 +1077,43 @@ def surface_list_from_polygons_and_gluings(polygons, gluings, mutable=False):
         s.set_immutable()
     return s
 
-####
```

Comment:
I removed these markers in the file and instead used the standard spacing "two newlines" between top-level entities.